app/resources/applications/ApplicationsController.jl

```diff
@@ -408,12 +418,41 @@ macro isonline(app)
   end
 end
 
+function move_to_folder(app, from_folder, to_folder)
+  app_path = joinpath(from_folder, app.name)
+
+  if !isdir(app_path)
+    @error "File '$app' is not a directory."
```

Comment:
Shouldn't this rethrow and break? This will just continue even if the dir does not exist...